patches/node-version.diff

```diff
@@ -35,7 +35,7 @@ Index: code-server/lib/vscode/build/lib/node.js
 -const version = /^target\s+"([^"]+)"$/m.exec(yarnrc)[1];
 +const version = process.versions.node;
  const platform = process.platform;
- const arch = platform === 'darwin' ? 'x64' : process.arch;
+ const arch = process.arch;
```

Comment:
Is this right?

Reply:
Looks like a change from upstream so yup!